crowdsourcer/management/commands/import_foi_data.py

```diff
@@ -0,0 +1,323 @@
+                {"description": "Green Tariff", "score": 1},
+                {"description": "Creates 20% own energy", "score": 1},
+                {"description": "Criteria not met", "score": 0},
+                {"description": "No response", "score": 0},
+            ]
+        },
+        "B&H Q3": {
+            "options": [
+                {"description": "50% or above", "score": 1},
+                {"description": "60% or above", "score": 2},
+                {"description": "90% or above", "score": 3},
+                {"description": "Criteria not met", "score": 0},
+                {"description": "No response", "score": 0},
+            ]
+        },
+    }
+
+    authority_name_map = {
+        "Lincoln City Council": "City of Lincoln",
+        "King's Lynn and West Norfolk Borough Council": "Kings Lynn and West Norfolk Borough Council",
+        "Common Council of the City of London": "City of London",
+        "London Borough of Barking and Dagenham Council": "Barking and Dagenham Borough Council",
+    }
+
+    def get_council_lookup(self):
+        url = get_dataset_url(
+            repo_name="uk_local_authority_names_and_codes",
+            package_name="uk_la_future",
+            version_name="1",
+            file_name="uk_local_authorities_future.csv",
+            done_survey=True,
+        )
+        df = pd.read_csv(url)
+
+        lookup = {}
+        for _, row in df.iterrows():
+            gss = row["gss-code"]
+            lookup[row["nice-name"].replace(" ", "-").lower()] = gss
+            lookup[row["nice-name"]] = gss
+            lookup[row["official-name"]] = gss
+
+        return lookup
+
+    def get_option_for_question(self, q, option):
+        option = Option.objects.get(question=q, description=option)
+
+        return option
+
+    def get_bh_3_answer(self, q, row, value):
+        if value == 0:
+            description = "Criteria not met"
+        elif value == 1:
+            try:
+                percentage = row.iloc(8)
+            except ValueError:
+                percentage = 50
+
+            threshold = 50
+            if percentage >= 90:
+                threshold = 90
+            elif percentage >= 60:
+                threshold = 60
+
+            description = f"{threshold}% or above"
+
+            option = Option.objects.get(question=q, description=description)
+
+            return option
+
+    def get_bh_2_answer(self, q, row, value):
+        options = []
+        if value == 0:
+            options.append(
+                Option.objects.get(question=q, description="Criteria not met")
+            )
+        elif value == 1:
+            options.append(
+                Option.objects.get(question=q, description="100% green energy")
+            )
+            if row.iloc[8] == 1:
+                options.append(
+                    Option.objects.get(question=q, description="Green Tariff")
+                )
+            if row.iloc[8] >= 20:
```

Comment:
I guess confused about what's happening between 1 and 20 here - is it not used?

Reply:
if the answer is less than 20% they don't get a point.

Reply:
But if row.iloc[8] is 1 they get "Green Tarriff" - what does that mean?

Reply:
oh, woops, the 8 is wrong, it should be nine.

Reply:
Hooray! I added value.

Reply:
This is, obviously, in no way an example of why using magic numbers is bad.